setup.py

```diff
@@ -23,7 +23,7 @@ def package_data(pkg, root_list):
 
 setup(
     name='xblock-drag-and-drop-v2',
-    version='2.0.16',
+    version='2.0.17',
```

Comment:
Note: This will removed before merge and a new Release PR will be created with this.